dev/scripts/src/alluxio.org/build-distribution/cmd/root.go

```diff
@@ -44,4 +45,5 @@ func additionalFlags(cmd *flag.FlagSet) {
 	cmd.BoolVar(&debugFlag, "debug", false, "whether to run this tool in debug mode to generate additional console output")
 	cmd.StringVar(&ufsModulesFlag, "ufs-modules", strings.Join(defaultModules(ufsModules), ","),
 		fmt.Sprintf("a comma-separated list of ufs modules to compile into the distribution tarball(s). Specify 'all' to build all ufs modules. Supported ufs modules: [%v]", strings.Join(validModules(ufsModules), ",")))
+	cmd.StringVar(&customUfsModuleFlag, "custom-ufs-module", "", "ufs-hadoop-x.x,hadoop-x.x,-pl underfs/hdfs -Pufs-hadoop-X -Dufs.hadoop.version=X.X.X")
```

Comment:
please describe the format in addition to providing an example. if we go with my suggestion of providing a name and maven args, then i would set the description string to be:
`"a comma-separated pair of the module name and its maven arguments. ex. hadoop-x.y,-pl underfs/hdfs -Pufs-hadoop-X -Dufs.hadoop.version=x.y.z"`

Reply:
Thanks